gitlib-3.1.1 build fails with conduit-1.3

Seen on the stackage build server; I was able to reproduce the build failure locally like so:

```bash
stack unpack gitlib-3.1.1 && cd gitlib-3.1.1
edit stack.yaml # add the following stack.yaml
stack build
```

```yaml
# stack.yaml
resolver: nightly-2018-03-10
extra-deps:
- conduit-1.3.0
- conduit-extra-1.3.0
- resourcet-1.2.0
```

Build failure:

```
[15 of 15] Compiling Git.Working      ( Git/Working.hs, .stack-work/dist/x86_64-osx/Cabal-2.0.1.0/build/Git/Working.o )

/Users/dan/scratch/gitlib-3.1.1/Git/Working.hs:18:33: error:
    Not in scope: type constructor or class ‘MonadBaseControl’
   |
18 | checkoutFiles :: (MonadGit r m, MonadBaseControl IO m, MonadIO m,
   |                                 ^^^^^^^^^^^^^^^^
```

```
Git/Working.hs:18:33: error:
    Not in scope: type constructor or class ‘MonadBaseControl’
   |
18 | checkoutFiles :: (MonadGit r m, MonadBaseControl IO m, MonadIO m,
   |                                 ^^^^^^^^^^^^^^^^
cabal: Leaving directory '/tmp/cabal-tmp-9373/gitlib-3.1.1'
cabal-2.2: Error: some packages failed to install:
gitlib-3.1.1-LABFhN4MmQjA42BknGMpOz failed during the building phase. The
exception was:
ExitFailure 1
```

Same here

--- a/gitlib/Git/Working.hs
+++ b/gitlib/Git/Working.hs
@@ -5,6 +5,7 @@ module Git.Working where
 import Conduit
 import Control.Applicative
 import Control.Monad.Catch
+import Control.Monad.Trans.Control
 import Data.Semigroup
 import Data.Text as T
 import Git.Blob

